examples/next/block-support/wp-blocks/index.js

```diff
@@ -12,4 +13,5 @@ export default {
   CoreButton: CoreBlocks.CoreButton,
   CoreButtons: CoreBlocks.CoreButtons,
   CoreHeading: CoreBlocks.CoreHeading,
+  CreateBlockBlockC: BlockC,
```

Comment:
Is there a reason we are not also including the A and B blocks as well?

Reply:
I feel that each example block should be a unique block type. Block A & Block B were originally added as a reference and are identical aside from their names.

I'm wondering if it would be better to only include 2 blocks for now; one using `registerFaustBlock` and the other using the traditional core function `registerBlockType`.

cc @theodesp

Reply:
@josephfusco Agreed, less confusing for the end user.

Reply:
Will update